### PR TITLE
Feat dockerised strip coverletter

### PR DESCRIPTION
**Pull request by lsh-0**
depends on: https://github.com/elifesciences/strip-coverletter/pull/7

### CODE REVIEW

File: salt/elife-bot/strip-coverletter.sls [hunk 1]
```diff
@@ -58,3 +39,19 @@ strip-coverletter-working:
...
             - install-strip-coverletter
+
+#
+# dockerised version
+#
+
+strip-coverletter-docker-image:
+    cmd.run:
+        - cwd: /opt/strip-coverletter
+        - name: ./build-image.sh
```
Comment (giorgiosironi) [inline salt/elife-bot/strip-coverletter.sls]: fine for now, but we shouldn't build the image every time, only pulling it from Docker Hub. Can be added later

Comment (giorgiosironi) [reply salt/elife-bot/strip-coverletter.sls]: may want to require `docker-ready` in these steps

Comment (lsh-0) [reply salt/elife-bot/strip-coverletter.sls]: I agree. It is very quick after first build however.

Comment (lsh-0) [reply salt/elife-bot/strip-coverletter.sls]: done

**Pull request merged**